src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -710,7 +733,9 @@ def get_devices(_sys_block_path='/sys/block', _dev_path='/dev', _mapper_path='/d
 
         metadata['partitions'] = get_partitions_facts(sysdir)
 
-        metadata['rotational'] = get_file_contents(sysdir + "/queue/rotational")
+        for key in ['rotational', 'nr_requests']:
```

Comment:
why is the `nr_requests` needed? doesn't seem like it is used anywhere else

Reply:
It's provisioning for later use. That could be a way to detect the good/bad configuration of a particular no-rotational drive.